modules/bastion/cloudinit.tf

```diff
@@ -6,6 +6,15 @@ data "cloudinit_config" "bastion" {
   gzip          = true
   base64_encode = true
 
+  part {
+    content_type = "text/cloud-config"
+    # https://cloudinit.readthedocs.io/en/latest/reference/examples.html#run-commands-on-first-boot
+    content = <<-EOT
+    runcmd:
+    - dnf config-manager --disable ol8_addons --disable ol8_appstream
```

Comment:
I wouldn't recommend disabling `ol8_appstream` as almost all installs will have RPMs from that repo. If you disable it, they will never be updated, i.e. you could be missing security fixes.